src/utils/wallet.js

```diff
@@ -406,22 +406,21 @@ class Wallet {
     async replaceAccessKey(oldKey, newKey) {
         const accountId = this.accountId;
         await this.getAccount(accountId).addKey(newKey)
-        await this.removeAccessKey(oldKey)
+        return await this.removeAccessKey(oldKey)
```

Comment:
@Patrick1904 why `return`? Not a blocker, just curious.

Reply:
Without it I don't get the success triggered when calling `sendNewRecoveryLink`. Maybe there's a different way?

![Screen Shot 2020-06-19 at 12 20 33 PM](https://user-images.githubusercontent.com/24921205/85175810-27d89800-b22d-11ea-9ec8-5db7472ee1ac.png)

Reply:
Got it – yeah, I think we need to change reducers to deal with `Promise<void>` results properly. I'll make a change and merge.

Reply:
Done here https://github.com/near/near-wallet/pull/681/commits/ac5970a0d89abc736b0401e8af231ff992be4546